docs/formats/point_cloud_user_manual.md

```diff
@@ -0,0 +1,40 @@
+# Point Cloud user manual
+
+## Contents
+
+- [Format specification](#format-specification)
+- [Load Point Cloud dataset](#load-point-cloud-dataset)
+
+## Format specification
+
+Point Cloud data format specification available [here](https://drive.google.com/file/d/1BtZyffWtWNR-mk_PHNPMnGgSlAkkQpBl/view).
```

Comment:
I see an archive by the link. Is it. expected?

Reply:
We used the link provided in SOW.